blazor/spreadsheet/getting-started-webapp.md

```diff
@@ -7,9 +7,9 @@ control: Spreadsheet
 documentation: ug
 ---
 
-# Getting Started with Blazor Spreadsheet in Blazor Web App
+# Getting Started with Blazor Spreadsheet in Web App
 
-This section briefly explains about how to include [Blazor Spreadsheet](https://www.syncfusion.com/blazor-components/blazor-pivot-table) component in your Blazor Web App using [Visual Studio](https://visualstudio.microsoft.com/vs/) and Visual Studio Code.
+This section briefly explains about how to include [Syncfusion Blazor Spreadsheet](https://www.syncfusion.com/blazor-components/blazor-spreadsheet) component in your Blazor Web App using [Visual Studio](https://visualstudio.microsoft.com/vs/) and Visual Studio Code.
```

Comment:
https://www.syncfusion.com/blazor-components

Use this URL to avoid URL link issue